### ISSUE TITLE
Components not keyboard accessible

### ISSUE THREAD

**Issue by twm** [appnexus/lucid#335]
I pursued the demos at http://appnexus.github.io/lucid/ a bit and found that few of the components are keyboard accessible.  For example, the [Expander](http://appnexus.github.io/lucid/#/components/Expander) widget cannot be focused, opened, or closed via keyboard.  The documentation isn't even navigable without using a mouse.

(I was evaluating the library and thought you might like to know why I cannot use it.)

**Comment by jondlm** [comment]
Thanks for the feedback! You're right that we haven't focussed on keyboard navigation for all of our components, but it's something we want to tackle.

`Expander` and `VerticalListMenu` could both definitely use work to improve keyboard navigation. If you think of any more, or have more specific interactions you're looking for, feel free to file bugs!
